tslint.json

```diff
@@ -7,6 +7,7 @@
         {
           "esSpecCompliant": true
         }
-      ]
+      ],
+      "quotemark": false
```

Comment:
why?

Reply:
@marlosin Because somehow we have different quotemark lint settings for the different projects which makes me crazy :)

Reply:
:cry: